spec/models/audit/event/check_spec.rb

```diff
@@ -53,9 +55,9 @@
 
     it 'produces the expected message' do
       expect(subject.message).to eq(
-        'rspec:user:my_user checked if rspec:host:my_host can execute ' \
-        'rspec:variable:my_var (failure)'
-      )
+                                   'rspec:user:my_user failed to check if rspec:host:my_host can execute ' \
```

Comment:
shouldn't this include also the error?

Reply:
in this test the checked role does not have permission to execute the resource, thats the "failure" of the test so there is no exception message to include